src/main/java/frc/robot/subsystems/pivot/PivotIOSim.java

```diff
@@ -56,7 +58,10 @@ public void setVoltage(double volts) {
   }
 
   @Override
-  public void setPivotAngle(double angle) {
-    pivotController.calculate(angle, pivotSim.getAngleRads());
+  public void setPivotAngle(double angleRots) {
```

Comment:
a comment or two would be nice in this method but that's nitpicking -- it is really simple so I get there being none